fusecli/cli/src/cmds/versions/version.rs

```diff
@@ -0,0 +1,65 @@
+// Copyright 2020-2021 The Datafuse Authors.
+//
+// SPDX-License-Identifier: Apache-2.0.
+
+use sha2::Digest;
+use sha2::Sha256;
+use structopt::StructOpt;
+use sysinfo::SystemExt;
+
+use crate::error::Result;
+
+#[derive(StructOpt, Debug)]
+pub struct VersionCommand {}
+
+impl VersionCommand {
```

Comment:
I think it should also support remote tag for version, i.e:
fusecli version --remote=true // show local client version and remote cluster version
fusecli version --remotve=false // only show local client version

Reply:
Good idea